cadc-log/src/test/java/ca/nrc/cadc/log/WebServiceLogInfoTest.java

```diff
@@ -160,6 +163,62 @@ public void testPathIsJobID()
         EasyMock.verify(request);
     }
 
+    @Test
+    public void testForwardedClientIP()
+    {
```

Comment:
Checkstyle--I see you have been updating the bracket style so it should be applied here too.